src/speedtest.cpp

```diff
@@ -0,0 +1,339 @@
+#include "speedtest.hpp"
+#include "position.hpp"
+#include "repetition_info.hpp"
+#include "search.hpp"
+#include "tm.hpp"
+#include "util/types.hpp"
+#include <iomanip>
+#include <iostream>
+#include <string>
+#include <vector>
+
+namespace Clockwork {
+namespace Speedtest {
+
+const std::vector<std::string> SPEEDTEST_FENS = {
```

Comment:
Could you instead pull some games from actual cw games? You can find many over at lichess.